LLMonFHIRUITests/FHIRDisplayTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+// This source file is part of the Stanford LLM on FHIR project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import XCTest
+
+
+final class FHIRDisplayTests: XCTestCase {
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+
+        continueAfterFailure = false
+
+        let app = XCUIApplication()
+        app.launchArguments = ["--skipOnboarding"]
+        app.deleteAndLaunch(withSpringboardAppName: "LLMonFHIR")
+    }
+
+    func testFHIRResourcesView() throws {
+        let app = XCUIApplication()
+        app.launchArguments = ["--skipOnboarding", "--testMode"]
```

Comment:
I would suggest to move these arguments to the `setupWithError` function. You can then remove this like and the launch command in the next line as the app is already lauschend from the setup call.